src/targets/cpu/include/migraphx/cpu/pointwise.hpp

```diff
@@ -43,7 +43,8 @@ struct multi_index
     {
```

Comment:
This `multi_index` struct is not used anywhere for the CPU. 

Should we just remove that ?

Reply:
Its used below.

Reply:
I don't find any reference when I search for its constructor or when i search for "multi_index"